<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_menu</class>
 <widget class="QDialog" name="main_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Main menu</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="reg_form_button">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Go to the registration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="log_in_form_button">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Go to login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exit_button">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
